Nodejs/Product/TypeScript/TypeScriptHelpers.cs

```diff
@@ -82,7 +82,20 @@ internal static string GetTypeScriptBackedJavaScriptFile(IVsProject project, str
 
             if (project is IVsBuildPropertyStorage props)
             {
-                ErrorHandler.ThrowOnFailure(props.GetPropertyValue(NodeProjectProperty.TypeScriptOutDir, null, (uint)_PersistStorageType.PST_PROJECT_FILE, out outDir));
+                // GetProperty can return this error code if the property doesn't exist
+                const int ERR_XML_ATTRIBUTE_NOT_FOUND = unchecked((int)0x8004C738);
+
+                try
+                {
+                    ErrorHandler.ThrowOnFailure(props.GetPropertyValue(NodeProjectProperty.TypeScriptOutDir, null, (uint)_PersistStorageType.PST_PROJECT_FILE, out outDir));
+                }
+                catch (System.Runtime.InteropServices.COMException e)
+                {
+                    if (e.ErrorCode == ERR_XML_ATTRIBUTE_NOT_FOUND)
```

Comment:
if the `ErrorCode` is not `ERR_XML_ATTRIBUTE_NOT_FOUND`, should it `throw`?

Reply:
Oops, that's what I meant to do. Good catch!